### PR TITLE
Add icons for key features

### PR DESCRIPTION
**Pull request by ee7**
The icons are now available, and `configlet lint` now requires that each key feature has a valid `icon` value.

Feel free to change these to whatever's preferred. The list of icons can be seen [here](https://github.com/exercism/docs/issues/189).

---

`powerful` Batteries Included
Epic & well-documented standard library. Need more? Use a PyPi package - there's one for everything.

`fun` Easy
Human-friendly Syntax and a vibrant, supportive community. Quick to learn & intuitive to use.

`extensible` Extensible
Need to call Fortran from a web API? Done. Need to process images using C? Python can do that.

`general-purpose` Flexible
Duck, dynamic, & strong typing. Easy to debug. Fun for experiments, robust for large applications.

`multi-paradigm` Multi-paradigm
OOP, structured, functional, & aspect-oriented. Adaptable to how you structure your programs.

`community` Ubiquitous
Accepted for almost any use. Easily interface with other languages & execute almost everywhere.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -3366,32 +3366,32 @@
     {
       "title": "Batteries Included",
       "content": "Epic & well-documented standard library. Need more? Use a PyPi package - there's one for everything.",
-      "icon": ""
+      "icon": "powerful"
```
Comment (ee7) [inline config.json:3369]: The icon for `powerful` might fit "Batteries Included" better than the icon for `productive`.

File: config.json [hunk 2]
```diff
@@ -3366,32 +3366,32 @@
...
     },
     {
       "title": "Easy",
       "content": "Human-friendly Syntax and a vibrant, supportive community. Quick to learn & intuitive to use.",
-      "icon": ""
+      "icon": "fun"
```
Comment (ee7) [inline config.json:3374]: `easy` is also available.

File: config.json [hunk 3]
```diff
@@ -3366,32 +3366,32 @@
...
       "title": "Multi-paradigm",
       "content": "OOP, structured, functional, & aspect-oriented. Adaptable to how you structure your programs.",
-      "icon": ""
+      "icon": "multi-paradigm"
     },
     {
       "title": "Ubiquitous",
       "content": "Accepted for almost any use. Easily interface with other languages & execute almost everywhere.",
-      "icon": ""
+      "icon": "community"
```
Comment (ee7) [inline config.json:3394]: You may or may not prefer the icon for `widely-used` here.

**Comment by BethanyG** [comment]
Hi @ee7  - thank you for submitting this.  I also submitted changes for icons (see PR #2477) - but had a brain fart of some sort and closed the PR.  I am re-opening that one and merging it now.  Apologies for the delay in seeing this, and for the work you went to.

**Pull request closed without merging**